addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -4262,6 +4268,31 @@ QUnit.test('mark channel as seen if last message is visible when switching chann
     );
 });
 
+QUnit.test('add custom filter should filter messages accordingly to selected filter', async function (assert) {
+    assert.expect(4);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        name: "General"
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'message_fetch') {
+                const domainsAsStr = args.kwargs.domain.map(domain => domain.join(''));
+                assert.step(`message_fetch_${domainsAsStr.join('')}`);
```

Comment:
would split `message_fetch` from the rest with another separator (`:`?) for clarity of assert, and maybe joint the rest with `,`?